File: multiConverter.js
// celsius, chf, fcfa

// var question = window.prompt("Do you want to convert a value ? (yes or no)")

function askToContinue(){
    userAnswer = window.prompt("Do you want to convert a value ? (yes or no)")
    if(userAnswer != "yes" && userAnswer != "no"){
        alert("Please enter a valid word. yes or no");}
    else if (userAnswer == "yes"){
        return true;
}   else if(userAnswer == "no"){   
        return false;
}}


function CelsiusConverter(){
    var celsius = window.prompt("Which value do you want to convert ?")
    try{
        if ((isNaN(celsius) || celsius == "")){
            throw "You have not enter a number. Try again..."}
        else{
        var celsiusNumber = Number(celsius);}
     }
    catch{
        console.log("Please enter a number");
     
        }
    var fahr = (celsiusNumber * 9/5) + 32;
    var message = "The result is " + fahr.toString() + " Farenheint!";
    return message;
    }
     
function chfConverter(){
    var chf = window.prompt("Which value do you want to convert ?")
    try{
        if ((isNaN(chf)) || chf == ""){
            console.log("Vous n\'avez pas entre un nombre")
        } else{
            var chfNumber = Number(chf)
        }
        
      }
    catch{
        console.log("You have not enter a number. Try again...");}
      
         
    var fcfa = chfNumber * 650;
    var message = "The result is " + fcfa.toString() + " FCFA!";
    return message;
     
     }
     
function fcfaConverter(){
    var fcfa = window.prompt("Which value do you want to convert ?")
    try{
        if ((isNaN(fcfa)) || fcfa == ""){
            console.log("You have not entered a number. Please try again...")}
        else{
        var fcfaNumber = Number(fcfa);
      }}
    catch{
        alert("You have not entered a number");
      
         }
    var chf = fcfaNumber / 650;
    var message = "The result is " + chf.toString() + " CHF!";
    return message;}

while (askToContinue()){

    // while (userAnswer != "yes" && userAnswer != "no"){
    // //     // var question = window.prompt("Do you want to convert a value ? (yes or no)")
    //     console.log("Please select a valid word between yes or no")}
    // // // var question = window.prompt("Do you want to convert a value ? (yes or no)")

    //     if (userAnswer== "no"){
    // //     // question = "no"
    //         console.log("Thank you!")}

    //     else if(userAnswer == "yes"){
    // //     // question = "yes"

    var userInput = window.prompt("Which conversion ? (CelsiusToFarenheit / chfToFcfa / fcfaToChf)");

        // while(userInput != "CelsiusToFarenheit" && userInput != "chfToFcfa "  && userInput != "fcfaToChf "){


        //     console.log("Please select a valid conversion.");
        //     userInput = window.prompt("Which conversion ? (CelsiusToFarenheit / chfToFcfa / fcfaToChf)")}
       
    if ( userInput == "chfToFcfa"){
            // userInput = "chfToFcfa"
        console.log(chfConverter());
            
    }
    else if ( userInput == "CelsiustoFarenheit"){
            // userInput = "CelsiustoFarenheit"

        console.log(CelsiusConverter());
           
    }
    
    else if (userInput = "fcfaToChf"){
            // userInput = "fcfaToChf"
        console.log(fcfaConverter());
            
            }
    else {
        console.log("Sorry you did not enter a valid conversion type.")
    }
}


  
    

// console.log("Thank you!");


    

    

//     var userInput = window.prompt("Which conversion ? (CelsiusToFarenheit / chfToFcfa / fcfaToChf)");

//     if (userInput == "CelsiustoFarenheit"){
//         console.log(CelsiusConverter());
//     }
    
//     else if ( userInput == "chfToFcfa"){
//         console.log(chfConverter());
//     }
    
//     else if (userInput == "fcfaToChf"){
//         console.log(fcfaConverter());
//     }

// }

// else{
//     console.log("Please select a word between yes or no ");
// }









// console.log("Thank you!!")

File: multi_converter.py
def chf_to_fcfa(chf):
    return chf * 650

def celsius_to_farenheit(celsius):
    return (celsius * 9/5) + 32

def fcfa_to_chf(fcfa):
    return fcfa / 650

def ask_to_continue():
    user_answer = input("Do you want to convert a value ? (yes or no) ")
    if user_answer == "yes" :
        return True
    else:
        return False

def conversion_type():
    user_answer = input("What do you want to convert ? (chf_to_fcfa / celsius_to_farenheit / fcfa_to_chf")
    return conversion_type

def ask_the_value():
    user_value = input("Enter the value you want to convert :")
    try:
        user_number = int(user_value)
    except:
        print("You did not enter a number.")
        quit()
    return user_value

while ask_to_continue():
    user_answer = conversion_type()
    user_number = ask_the_value()
    result_message = "The result is "


    if user_answer == "chf_to_fcfa":
        result_message += str(chf_to_fcfa(user_number)) + " FCFA."

    elif user_answer == "celsius_to_farenheit":
        result_message += str(celsius_to_farenheit(user_number)) + " Farenheit."

    elif user_answer == "fcfa_to_chf":
        result_message += str(fcfa_to_chf(user_number)) + " CHF."

    else:
        result_message = " You did not enter a valid conversion type."

    print(result_message)
